ndcube/utils/cube.py

```diff
@@ -138,8 +141,16 @@ def get_crop_item_from_points(points, wcs, crop_by_values, keepdims):
     # Define a list of lists to hold the array indices of the points
     # where each inner list gives the index of all points for that array axis.
     combined_points_array_idx = [[]] * wcs.pixel_n_dim
+    high_level_wcs = HighLevelWCSWrapper(wcs) if isinstance(wcs, BaseLowLevelWCS) else wcs
+    wcs = high_level_wcs.low_level_wcs
     # For each point compute the corresponding array indices.
     for point in points:
+        # Sanitize input format
+        # Make point a tuple if given as a single high level coord object valid for this WCS.
+        if isinstance(point, tuple(v[0] for v in wcs.world_axis_object_classes.values())):
```

Comment:
*activates APE-14 lawyer mode*

This is technically invalid if `wcs.seralized_classes is True`, as the first element of `world_axis_object_classes` will be a string. I've never actually seen a WCS in the wild which uses `seralized_classes` so we could just throw an error here if it's True?